experimental/dds/tree2/src/class-tree/tree.ts

```diff
@@ -0,0 +1,116 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IChannel } from "@fluidframework/datastore-definitions";
+import { ISubscribable } from "../events";
+import { IDisposable, disposeSymbol } from "../util";
+import { FlexTreeView, type CheckoutEvents } from "../shared-tree";
+import { getProxyForField, type Unhydrated } from "../simple-tree";
+import { TreeFieldSchema as FlexTreeFieldSchema } from "../feature-libraries";
+import { NodeFromSchema, TreeNodeSchema } from "./schemaTypes";
+/**
+ * Channel for a Tree DDS.
+ * @alpha
+ */
+export interface ITree extends IChannel {
+	/**
+	 * Returns a tree known to be compatible with the provided schema with a schema aware API based on that schema.
```

Comment:
This is a bit hard to parse (lots of "schema"). Maybe something like:
```suggestion
	 * Returns a tree that is known to be compatible with the provided schema. The returned tree exposes an API that is schema-aware.
```